src/views/Home/Home.js

```diff
@@ -84,7 +87,7 @@ function Home() {
                 <div
                   className="col s12"
                   onClick={() => {
-                    playAudio(welcome_ta, "ta");
+                    playAudio(welcome_hi, myCurrectLanguage);
```

Comment:
Please check this line why it is showing welcome_hi if we are getting language setting language default

Reply:
It just a welcome audio. We are not using it anywhere. it's from proto1 and we only have Cat and I want to learn about mesurement in proto 1 not else.
If you know then please tell me